test/publish_test.js

```diff
@@ -4,58 +4,6 @@ suite("api/publish", function() {
   var aws             = require('aws-sdk');
   var assert          = require('assert');
   var Promise         = require('promise');
-  var mockAuthServer  = require('taskcluster-lib-testing/.test/mockauthserver');
-
-  // Sanity checks for the reference
-  test("reference from mockAuthServer", function() {
-    var reference = mockAuthServer.api.reference({
-      baseUrl:          'http://localhost:23243/v1'
-    });
-    assert(reference.entries, "Missing entries");
-    assert(reference.entries.length > 0, "Has no entries");
-    assert(reference.title, "Missing title");
-  });
-
-  // Test publish reference from mockAuthServer
-  test("publish reference from mockAuthServer", function() {
-    var cfg = config({
-      envs: [
-        'aws_accessKeyId',
-        'aws_secretAccessKey',
-        'aws_region',
-        'aws_apiVersion',
-        'referenceTestBucket'
-      ],
-      filename:               'taskcluster-base-test'
-    });
-
-    if (!cfg.get('aws') || !cfg.get('referenceTestBucket')) {
-      throw new Error("Skipping 'publish', missing config file: " +
-                      "taskcluster-base-test.conf.json");
-      return;
-    }
-
-    return mockAuthServer.api.publish({
-      baseUrl:              'http://localhost:23243/v1',
-      referencePrefix:      'base/test/api.json',
-      referenceBucket:      cfg.get('referenceTestBucket'),
-      aws:                  cfg.get('aws')
-    }).then(function() {
-      // Get the file... we don't bother checking the contents this is good
-      // enough
-      var s3 = new aws.S3(cfg.get('aws'));
-      return s3.getObject({
-        Bucket:     cfg.get('referenceTestBucket'),
-        Key:        'base/test/api.json'
-      }).promise();
-    }).then(function(res) {
-      var reference = JSON.parse(res.Body);
-      assert(reference.entries, "Missing entries");
-      assert(reference.entries.length > 0, "Has no entries");
-      assert(reference.title, "Missing title");
-    });
-  });
-
```

Comment:
Agreed - 'publish minimal reference', below, does the same thing without co-opting the mockAuthServer's API definition.